android/app/src/main/java/app/organicmaps/util/Utils.java

```diff
@@ -662,6 +662,21 @@ public static String getLocalizedBrand(@NonNull Context context, @Nullable Strin
     return brand;
   }
 
+  public static String getLocalizedLevel(@NonNull Context context, @Nullable String level) {
+    if (TextUtils.isEmpty(level) || level == null) {
+      return "";
+    }
+    if ("0".equals(level)) {
+      return context.getString(R.string.level_value_0);
+    }
+    try {
+      int parsedLevel = Integer.valueOf(level);
+      return context.getString(R.string.level_value_generic, parsedLevel);
+    } catch(NumberFormatException e) {
+      return level;
```

Comment:
if level is non-integer, e.g. `0;1` or `0.5` then it should be displayed like `Level: 0.5` still, not a bare `0.5` string

Reply:
I see two ways of doing that:
* give up on number parsing altogether and use `%s` instead of `%d` in the messages. Downside: no plurals support
* introduce a third message id (say `level_non_string_value`) for cases where we can't parse the string as a number, which uses `%s` as a placeholder. Downside: it's one more message id

Which one would you go for?

Reply:
My understanding is that we don't need plurals support here.
As the text is referencing a single specific level number, but its not about "a number of levels".

IMO we can just replace with `localized_level_string + ": " + level_value`, WDYT?

Reply:
> IMO we can just replace with localized_level_string + ": " + level_value, WDYT?

That's generally something I would advise against as a bad practice in localization. Not all languages work like this.
For instance, I just noticed that I messed up the French localization: it should be "Niveau : %d", not "Niveau: %d".
Surely you can find other examples of languages where it works differently (right-to-left scripts for instance?)

Reply:
Sure we can keep the placeholder then.
Are you aware of any languages that might need plurals support for this case though?

Reply:
I have removed the plurals support.